charts/argo-cd/templates/NOTES.txt

```diff
@@ -15,8 +15,14 @@ In order to access the server UI you have the following options:
       - Add the `--insecure` flag to `server.extraArgs` in the values file and terminate SSL at your ingress: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/ingress.md#option-2-multiple-ingress-objects-and-hosts
 
 
+{{ if eq (index .Values.server.config "admin.enabled") "true" -}}
 After reaching the UI the first time you can login with username: admin and the random password generated during the installation. You can find the password by running:
 
 kubectl -n {{ .Release.Namespace }} get secret argocd-initial-admin-secret -o jsonpath="{.data.password}" | base64 -d
 
 (You should delete the initial secret afterwards as suggested by the Getting Started Guide: https://github.com/argoproj/argo-cd/blob/master/docs/getting_started.md#4-login-using-the-cli)
+{{ else if index .Values.server.config "oidc.config" -}}
```

Comment:
Maybe lets also check for `dex.config`?

Reply:
Resolved in 73cdb48e1aa2c6a47f45e23708ee74ce9d1dc9ab